src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.4.td.json

```diff
@@ -0,0 +1,74 @@
+{
+  "_guid_" : "S-131.4",
+  "_extends_" : "F-131_Test_Data_Base",
+
+  "specs" : [
+    "with no R access for a field being created",
+    "attempts to create an event for the previously created case",
+    "does not display the newly created text field"
+  ],
+
+  "request": {
+    "_extends_": "Common_Request",
+    "pathVariables": {
+      "caseId": "${[scenarioContext][childContexts][F-131_CreateCase][testData][actualResponse][body][id]}"
+    },
+
+    "headers": {
+      "experimental": true
```

Comment:
we can remove this line as it's in the base file

Reply:
Removed this line as it's in the base file